src/java.base/share/conf/security/java.security

```diff
@@ -304,6 +304,42 @@ keystore.type=pkcs12
 #
 fips.keystore.type=pkcs12
 
+#
+# Location of the NSS DB keystore (PKCS11) in FIPS mode.
+#
+# The syntax for this property is identical to the 'nssSecmodDirectory'
+# attribute available in the SunPKCS11 NSS configuration file. Use the
+# 'sql:' prefix to refer to an SQLite DB.
+#
+# If the system property fips.nssdb.path is also specified, it supersedes
+# the security property value defined here.
+#
+# Note: the default value for this property points to an NSS DB that might be
+# readable by multiple operating system users and unsuitable to store keys.
+#
+fips.nssdb.path=sql:/etc/pki/nssdb
+
+#
+# PIN for the NSS DB keystore (PKCS11) in FIPS mode.
+#
+# Values must take any of the following forms:
+#   1) pin:<value>
+#        Value: clear text PIN value.
+#   2) env:<value>
+#        Value: environment variable containing the PIN value.
+#   3) file:<value>
+#        Value: path to a file containing the PIN value in its first
+#        line.
+#
```

Comment:
The prefixes are listed in lower-case here, but the checks are case-sensitive and all caps.
We should either fix the documentation or make the check case-insensitive.

Reply:
While the checks are case-sensitive, we make the obtained prefix uppercase  [here](https://github.com/rh-openjdk/jdk/pull/17/files#diff-dfa0dff6275fe003a6a2b68903fe3b3d9b0fac72d988f50a3554cd533c9efbd9R75), before any comparison. Regarding documenting the prefix as lower or uppercase, I prefer lowercase for better alignment with NSS tools. The current `NssdbPin.java` test should be exercising this because it's passing lowercase, as documented for users.

Reply:
I made the change to lowercase anyways, for better alignment between the code and the doc. Thanks.

Reply:
Thanks, I missed the case conversion, which is more efficient than the `equalsIgnoreCase` I was thinking of. Thanks for syncing the docs with the code.